packages/ui-view/src/ContextView/__tests__/ContextView.test.js

```diff
@@ -42,74 +40,4 @@ describe('<ContextView />', async () => {
     const contextView = within(subject.getDOMNode())
     expect(await contextView.accessible()).to.be.true()
   })
-
-  it('should apply default styled arrow by default', async () => {
-    const subject = await mount(<ContextView />)
-
-    const contextView = within(subject.getDOMNode())
-    const arrow = await contextView.find(`.${styles['arrow']}`)
-
-    expect(arrow.hasClass(styles['arrow--default'])).to.be.true()
-  })
-
-  it('should apply inverse arrow when inverse is set', async () => {
-    const subject = await mount(<ContextView background="inverse" />)
-
-    const contextView = within(subject.getDOMNode())
-    const arrow = await contextView.find(`.${styles['arrow']}`)
-
-    expect(arrow.hasClass(styles['arrow--inverse'])).to.be.true()
-  })
-
-  function testPlacement(placement) {
-    it(`should apply placement classes for ${placement}`, async () => {
-      const subject = await mount(<ContextView placement={placement} />)
-      const classname = styles[`placement--${placement.split(' ').join('-')}`]
-      const contextView = within(subject.getDOMNode())
-
-      expect(contextView.hasClass(classname)).to.be.true()
-    })
-  }
-
-  testPlacement('top')
-  testPlacement('top start')
-  testPlacement('top end')
-  testPlacement('start')
-  testPlacement('start top')
-  testPlacement('start bottom')
-  testPlacement('bottom')
-  testPlacement('bottom start')
-  testPlacement('bottom end')
-  testPlacement('end')
-  testPlacement('end top')
-  testPlacement('end bottom')
-  testPlacement('offscreen')
-
-  function testArrowPlacement(placement, mirror) {
-    it(`should mirror the arrow position based on the placement for ${placement}`, async () => {
-      const subject = await mount(
-        <ContextView placement={placement} padding="small">
-          Hello World
-        </ContextView>
-      )
-      const classname = styles[`arrow--${mirror.split(' ').join('-')}`]
-      const contextView = within(subject.getDOMNode())
-      const arrow = await contextView.find(`.${styles['arrow']}`)
-
-      expect(arrow.hasClass(classname)).to.be.true()
-    })
-  }
-
-  testArrowPlacement('top', 'bottom')
-  testArrowPlacement('top start', 'bottom start')
-  testArrowPlacement('top end', 'bottom end')
-  testArrowPlacement('start', 'end')
-  testArrowPlacement('start top', 'end top')
-  testArrowPlacement('start bottom', 'end bottom')
-  testArrowPlacement('bottom', 'top')
-  testArrowPlacement('bottom start', 'top start')
-  testArrowPlacement('bottom end', 'top end')
-  testArrowPlacement('end', 'start')
-  testArrowPlacement('end top', 'start top')
-  testArrowPlacement('end bottom', 'start bottom')
 })
```

Comment:
Note: these test cases were deleted because with the current emotion styling solution we simply can't query for css classes like we could do with the previous solution.